src/views/Header.js

```diff
@@ -80,6 +83,24 @@ class Header extends React.Component<void, HeaderProps, void> {
     );
   }
 
+  _tvEventHandler: TVEventHandler;
+
+  componentDidMount(): void {
+    this._tvEventHandler = new TVEventHandler();
+    this._tvEventHandler.enable(this, function(cmp, evt) {
+      if (evt && evt.eventType === 'menu') {
+        cmp.props.onNavigateBack && cmp.props.onNavigateBack();
```

Comment:
If you use arrow function instead of `function` keyword, you can just do `this.props.onNavigateBack`